scripts/codegen-utils.js

```diff
@@ -0,0 +1,124 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+const packageJSON = require('../package.json');
+
+const ERROR_PREFIX = 'react-native-gesture-handler';
```

Comment:
not this lib

Reply:
fixed in: https://github.com/software-mansion/react-native-svg/pull/2357/commits/b20a8c8157363c41523bf15b1ff480bc0ad133e3